File: FlightProject/src/Entity/FlightDetails.java
package Entity;

//flight details is used similarly to a data structure for easier marshaling and passing of the following fields
public class FlightDetails {
	//departure time of flight
	private long time;
	//airfare of flight
	private float airfare;
	//number of vacant seats remaining for flight
	private int availableSeats;
	
	//getter and setter of attributes
	///////////////////////////////////////////////////
	public long getTime() {
		return time;
	}
	public void setTime(long time) {
		this.time = time;
	}
	public float getAirfare() {
		return airfare;
	}
	public void setAirfare(float airfare) {
		this.airfare = airfare;
	}
	public int getAvailableSeats() {
		return availableSeats;
	}
	public void setAvailableSeats(int availableSeats) {
		this.availableSeats = availableSeats;
	}
	///////////////////////////////////////////////////
}

File: PyFlightProject/client/FlightDetails.py
'''
Created on 2 Apr, 2015

@author: Ronald
'''

#python version of the java Flight Details Class
class FlightDetails(object):
    '''
    classdocs
    '''
    def __init__(self, time=0, airfare=0.0, available_seats=0):
        '''
        Constructor
        '''
        self.time = time
        self.airfare = airfare
        self.available_seats = available_seats
    
    def get_time(self):
        return self.time
    
    def set_time(self, time):
        self.time = time
        
    def get_airfare(self):
        return self.airfare
    
    def set_airfare(self, airfare):
        self.airfare = airfare
        
    def get_available_seats(self):
        return self.available_seats
    
    def set_available_seats(self, available_seats):
        self.available_seats = available_seats  